processing/data_collection/gazette/settings.py

```diff
@@ -9,9 +9,8 @@
     "spidermon.contrib.scrapy.pipelines.ItemValidationPipeline": 400,
     "gazette.pipelines.SQLDatabasePipeline": 500,
 }
-
+QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
```

Comment:
Since the text extraction was removed from the repository, this setting does not exist anymore.